test/commit-validator.spec.js

```diff
@@ -6,69 +6,75 @@ describe('CommitValidator', () => {
     scopes: ['foo', 'bar'],
   };
 
-  it('returns false if message does not contain valid task', () => {
+  it('returns false if message does not contain valid task', async () => {
     const validator = new CommitValidator(options);
 
-    expect(validator.isValid('build(foo): JIRA-1234 Add build script')).toBe(false);
-    expect(validator.isValid('docs(bar): JIRA-1234 Update docs')).toBe(false);
+    expect((await validator.isValid('build(foo): JIRA-1234 Add build script')).valid).toBe(false);
```

Comment:
I know this might not be the best syntax/approach... but I want to hold off on refactoring the tests for a later date. For this change, I just wanted to ensure they validated correctly